### ISSUE TITLE
Mac OS gcloud app deploy failed

### ISSUE THREAD

**Issue by geogismx** [KMarkert/mekong-wq-gae#1]
Error: Server Error
The server encountered an error and could not complete your request.
Please try again in 30 seconds.

**Comment by eltonhcchan** [comment]
It seems the server has not been working since 2019. what happened?

**Comment by KMarkert** [comment]
Yes, the application has not worked for deployment for some time. This is due to the update to the [earthengine-api](https://github.com/google/earthengine-api) where old API calls will not work on their servers. This will require the [earthengine package in the application code](https://github.com/KMarkert/mekong-ssc-gae/tree/master/lib/ee) to be replace with the latest version.

There may be other issues (like api calls) that could be affecting the app but this is a good start. It is on my to-do list to get to.
